querybook/server/tasks/run_datadoc.py

```diff
@@ -149,14 +153,54 @@ def _start_query_execution_task(
         return query_execution.id
 
 
+@with_session
+def get_datadoc_error_message(query_execution_id, data_cell_id=None, session=None):
+    if not data_cell_id:
+        _, data_cell_id = qe_logic.get_datadoc_id_from_query_execution_id(
```

Comment:
sorry for my last comment, I just realized I made a mistake because you need the previous cell id, not the current one